Makefile

```diff
@@ -71,7 +71,7 @@ build-testkube-bin-intel:
 		cmd/kubectl-testkube/main.go
 
 docker-build-api:
-	docker build -t api-server -f build/api-server/Dockerfile .
+	docker build  --platform linux/x86_64 -t kubeshop/testkube-api-server:$(COMMIT)-dev -f build/api-server/Dockerfile .
```

Comment:
Is this for testing?

Reply:
yes for local builds to share with someone ;)